test/integration/krane_test.rb

```diff
@@ -38,19 +36,15 @@ def test_render_black_box
     bindings = "data=#{data_value}"
     test_sha = rand(10_000).to_s
 
-    out, err, status = nil
-    with_env("REVISION", test_sha) do
-      out, err, status = krane_black_box("render", "-f #{paths.join(' ')} --bindings #{bindings}")
-    end
+    out, err, status = krane_black_box("render", "-f #{paths.join(' ')} --bindings #{bindings} "\
```

Comment:
should we just stick the second arg on its own line rather than split the string